### PR TITLE
Re-add RJIT::Instruction#opes

### PR DESCRIPTION
**Pull request by jhawthorn**
Useful for "bring your own JIT" implementations to have type information of arguments

I left it as `nil` for trace instructions.

cc @k0kubun @tenderlove

### CODE REVIEW

**Review by k0kubun — APPROVED** [review]

**Comment by ioquatix** [comment]
What is `opes`? It looks like `oops` or `ops` but I'm not sure what `opes` is. Can we expand the abbreviation or add documentation?

**Comment by tenderlove** [comment]
> What is `opes`? It looks like `oops` or `ops` but I'm not sure what `opes` is. Can we expand the abbreviation or add documentation?

It means "operands", and I think it comes from the [VM generation scripts](https://github.com/ruby/ruby/blob/master/tool/ruby_vm/models/bare_instructions.rb#L19).  Unsure why it's shortened that way though.

Maybe we should change it to `operands`? 🤷‍♀️

**Review by tenderlove — APPROVED** [review]

**Comment by k0kubun** [comment]
I use the name `operand` in RJIT, so feel free to rename it to `operands`. If you do that, though, I'd like you to rename [the original `opes`](https://github.com/ruby/ruby/blob/152ccbe859f5c6411c6342568b717943f157927c/tool/ruby_vm/models/bare_instructions.rb#L19) as well.

**Comment by ioquatix** [comment]
+1 to renaming it to something meaningful.

**Comment by jhawthorn** [comment]
I don't feel strongly (though I think "opnd" is a more common abbreviation of "operand" than "ope"), but added the rename for BareInstruction as well.

**Review by k0kubun — APPROVED** [review]

File: tool/ruby_vm/views/lib/ruby_vm/rjit/instruction.rb.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+  Instruction = Data.define(:name, :bin, :len, :operands)
 
   INSNS = {
 % RubyVM::Instructions.each_with_index do |insn, i|
     <%= i %> => Instruction.new(
       name: :<%= insn.name %>,
       bin: <%= i %>, # BIN(<%= insn.name %>)
       len: <%= insn.width %>, # insn_len
-      opes: <%= (insn.opes unless insn.name.start_with?('trace_')).inspect %>,
+      operands: <%= (insn.opes unless insn.name.start_with?('trace_')).inspect %>,
```
Comment (ioquatix) [inline tool/ruby_vm/views/lib/ruby_vm/rjit/instruction.rb.erb]: So happy to see this.

**Pull request merged**